projects/plugins/jetpack/modules/wordads/php/class-wordads-smart.php

```diff
@@ -310,11 +291,44 @@ private function get_language_keywords(): array {
 	}
 
 	/**
-	 * Gets the blog's ID.
+	 * Enable formats by post types and the display options.
+	 *
+	 * @return void
+	 */
+	private function enable_formats(): void {
+		$this->formats['top']['enabled']                  = $this->params->options['enable_header_ad'];
+		$this->formats['inline']['enabled']               = is_singular( 'post' ) && $this->params->options['wordads_inline_enabled'];
+		$this->formats['belowpost']['enabled']            = $this->params->options['wordads_second_belowpost'];
```

Comment:
I don't think we want to map the `belowpost` format to the `wordads_second_belowpost` option. It removes the consideration for default below post. _I think we should leave `belowpost2` for now since we are yet to find any meaningful indication of its current use based on uslicer stats._

Maybe we can use `$this->params->should_show()`. Since it correctly checks for the combination of `wordads_display_front_page`, `wordads_display_post`, `wordads_display_page`, and `wordads_display_archive` with respect to the toggle messaging for belowpost on the Ad settings.
<img width="386" alt="below post option" src="https://github.com/user-attachments/assets/91952a13-8419-46a2-a1a5-5fe9c62f7ef8">

Reply:
Good idea!